tracer/src/Datadog.Trace/RemoteConfigurationManagement/Json/TufRootBase64Converter.cs

```diff
@@ -19,9 +20,12 @@ public override TufRoot ReadJson(JsonReader reader, Type objectType, TufRoot exi
                 return null;
             }
 
-            var contentDecode = Encoding.UTF8.GetString(Convert.FromBase64String((string)reader.Value));
+            var contentDecode = Convert.FromBase64String((string)reader.Value);
```

Comment:
I'm not sure about this, but would using an intermediary `CryptoStream` as suggested on https://stackoverflow.com/a/35795205 instead of calling `FromBase64String` save us this allocation (which in the case of ASM's 240kb payload, will end up on the LOH)?

Reply:
Nice.
Changed here 8628e083933aa08900d2ae5d323dc3e799a9c91c

Reply:
Actually, it fails now on `AssemblyReferencesHaveNotChanged` test, so I'm reverting commit 😢